policyengine_canada/variables/household/income/taxable_other_than_dividend_income.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class taxable_other_than_dividend_income(Variable):
```

Comment:
Is this all taxable income except dividends? If so I'd just do that subtraction in the formula